apps/central-scan/frontend/src/components/delete_batch_modal.test.tsx

```diff
@@ -0,0 +1,82 @@
+import React from 'react';
+import { QueryClient, QueryClientProvider } from '@tanstack/react-query';
+import userEvent from '@testing-library/user-event';
+import { suppressingConsoleOutput } from '@votingworks/test-utils';
+import { MockApiClient, createMockApiClient } from '../../test/api';
+import { render, screen, waitFor } from '../../test/react_testing_library';
+import { ApiClientContext } from '../api';
+import { DeleteBatchModal } from './delete_batch_modal';
+
+let mockApiClient: MockApiClient;
+
+beforeEach(() => {
+  jest.restoreAllMocks();
+  mockApiClient = createMockApiClient();
+});
+
+afterEach(() => {
+  mockApiClient.assertComplete();
+});
+
+test('allows canceling', async () => {
+  const onClose = jest.fn();
+
+  render(
+    <QueryClientProvider client={new QueryClient()}>
```

Comment:
It's generally good to use the same query client options in tests as in the app, since they can influence the logic significantly.